cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -548,7 +548,7 @@ bundle agent cfe_internal_clear_last_seen_hosts_logs
 
   commands:
 
-    (policy_server|am_policy_hub).enterprise_edition.active_hub::
+    (policy_server|am_policy_hub).enterprise_edition.active_hub.cfengine_3_12.cfengine_3_13.cfengine_3_14::
```

Comment:
Oh, now this won't ever run.

It needs to be:

```
     (policy_server|am_policy_hub).enterprise_edition.active_hub.(cfengine_3_12|cfengine_3_13|cfengine_3_14)::
```

In order to run on enterprise hubs that are active and running version 3.12.x, 3.13.x or 3.14.x